src/Tribe/Assets.php

```diff
@@ -75,9 +75,10 @@ public function admin_enqueue_scripts() {
 					array( 'event-tickets-admin-css', 'tickets.css' ),
 					array( 'event-tickets-admin-refresh-css', 'tickets-refresh.css', array( 'event-tickets-admin-css' ) ),
 					array( 'event-tickets-admin-tables-css', 'tickets-tables.css', array( 'event-tickets-admin-css' ) ),
-					array( 'event-tickets-admin-js', 'tickets.js', array( 'jquery-ui-datepicker tribe-bumpdown' ) ),
-					array( 'event-tickets-admin-tables-js', 'tickets-tables.js', array( 'event-tickets-admin-js' ) ),
-					array( 'event-tickets-admin-accordion-js', 'accordion.js', array( 'event-tickets-admin-js' ) ),
+					array( 'event-tickets-admin-js', 'tickets.js', array( 'jquery-ui-datepicker' ) ),
+					array( 'event-tickets-admin-tables-js', 'tickets-tables.js', array( 'underscore' ) ),
+					array( 'event-tickets-admin-accordion-js', 'accordion.js', array() ),
+					array( 'tribe-bumpdown', 'bumpdown.js', array( 'jquery', 'underscore', 'hoverIntent' ) ),
```

Comment:
we don't need to re-register `tribe-bumpdown` we need just to add it as dependency on the any of the other scripts.